src/consumer.cpp

```diff
@@ -93,7 +96,7 @@ Consumer::consume(const Name& dataName,
                   const ErrorCallback& errorCallback)
 {
   Interest interest(dataName);
-  interest.setMustBeFresh(true);
+  // interest.setMustBeFresh(true);
```

Comment:
Could you write comments inline to explain why this flag should not be set?

Reply:
I can, but first, we need justification on why it was/ or should be MustBeFresh. Most of the applications don't want to sever just the fresh data, so what's the reason for having it mustbefresh in the first place.

Reply:
I can see two approaches:
1/ Easy fix: Removing this line and leaving a comment to explain the tradeoff
2/ Longer term: letting user to specify Interest parameters.

Either is fine to me.